app/src/main/java/tech/relaycorp/gateway/domain/endpoint/NotifyEndpoint.kt

```diff
@@ -19,8 +19,7 @@ class NotifyEndpoint
             ?.let { endpoint ->
                 context.sendBroadcast(
                     Intent(RelaynetAndroid.ENDPOINT_NOTIFY_ACTION)
-                        .setPackage(endpoint.applicationId),
-                    RelaynetAndroid.GATEWAY_SYNC_PERMISSION
+                        .setPackage(endpoint.applicationId)
```

Comment:
From a search on GH, it seems like `GATEWAY_SYNC_PERMISSION` isn't used anywhere else. Can you please delete it?